code/modules/jobs/job_types/cook.dm

```diff
@@ -43,6 +44,9 @@
 		if(!visualsOnly)
 			J.cooks++
 
+/datum/outfit/job/cook/equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+
```

Comment:
whats the point of this?

Reply:
Vestige code from trying to figure out why cook wasn't getting his chip